src/inputs.ts

```diff
@@ -45,13 +45,15 @@ export class ActionConfig {
   public readonly commitDescription: string;
   public readonly commitTitle: string;
   public readonly enableComments: boolean;
+  public readonly ignoredGlob: string;
   public readonly isDryRun: boolean;
   public readonly svgoOptionsPath: string;
 
   constructor(config: RawActionConfig = { }) {
     this.commitDescription = ActionConfig.getCommitDescription(config);
     this.commitTitle = ActionConfig.getCommitTitle(config);
     this.enableComments = ActionConfig.getCommentsValue(config);
+    this.ignoredGlob = "";
```

Comment:
Add TODO comment to clarify why this is an empty string

```suggestion
    this.ignoredGlob = ""; // TODO: make configurable, see #162
```